src/js/views/eva-multi-variant-view.js

```diff
@@ -0,0 +1,176 @@
+                                variant[0].associatedRSID + '&species=' + species + '">' +
+                                variant[0].associatedRSID + '</a>';
+            }
+
+            var detailedViewURLForAccession = '<a href="' + this.detailedViewURL.replace("<ACCESSION_ID>", accessionID)
+                                                + '">Detailed view</a>';
+            summaryData = variant.map(function(x) {
+                var summaryDataObj = {};
+                summaryDataObj[summaryDisplayFields.id] = x.id;
+                summaryDataObj[summaryDisplayFields.assoc_rs] = rsReference;
+                summaryDataObj[summaryDisplayFields.detailedViewURLLink] = detailedViewURLForAccession;
+                summaryDataObj[summaryDisplayFields.organism] = organism;
+                summaryDataObj[summaryDisplayFields.assembly] = assemblyLink;
+                summaryDataObj[summaryDisplayFields.submitterHandle] = x.submitterHandle;
+                summaryDataObj[summaryDisplayFields.contig] = x.contig;
+                summaryDataObj[summaryDisplayFields.chromosome] = x.chromosome;
+                summaryDataObj[summaryDisplayFields.start] = x.start;
+                summaryDataObj[summaryDisplayFields.end] = x.end;
+                summaryDataObj[summaryDisplayFields.reference] = _.escape(x.referenceRepr);
+                summaryDataObj[summaryDisplayFields.alternate] = _.escape(x.alternateRepr);
+                summaryDataObj[summaryDisplayFields.id] = x.id;
+                summaryDataObj[summaryDisplayFields.type] = x.variantTypeLink;
+                summaryDataObj[summaryDisplayFields.evidence] = x.evidence;
+                summaryDataObj[summaryDisplayFields.assemblyMatch] = x.assemblyMatch;
+                summaryDataObj[summaryDisplayFields.allelesMatch] = x.allelesMatch;
+                summaryDataObj[summaryDisplayFields.validated] = x.validated;
+                summaryDataObj[summaryDisplayFields.createdDate] = x.createdDate;
+                return summaryDataObj;
+            });
+        }
+        else {
+            var summaryDataEntry = {};
+            _.each(_.values(summaryDisplayFields), function(key) {
+                if (key == summaryDisplayFields.id) {
+                    summaryDataEntry[key] = "<b>No data available for " + accessionID + "</b>";
+                }
+                else {
+                    summaryDataEntry[key] = " ";
+                }
+            });
+            summaryData = [summaryDataEntry];
+        }
+
+        return summaryData;
+    },
+
+    _getPagerContent: function () {
+        var navURLTemplate = window.location.origin + window.location.pathname
+                             + "?variant&accessionID=" + this.accessionIDs.join(",")
+                             + "&species=" + this.species
+                             + "&page=<PAGE>";
+        var prevLink = (this.page > 1) ? navURLTemplate.replace("<PAGE>", "" + (this.page - 1)): "";
```

Comment:
same question as in the other replace. You can even make a local function here if you want to keep it near this place.